stablehlo/dialect/StablehloOps.cpp

```diff
@@ -178,36 +178,44 @@ Value maybeCastTo(OpBuilder& b, Location loc, Value value, Type type) {
 //===----------------------------------------------------------------------===//
 
 // Convert a 1D dense int64 attribute to a list of values.
-SmallVector<int64_t> convertDenseIntAttr(
-    llvm::Optional<mlir::DenseIntElementsAttr> optionalAttr) {
+FailureOr<SmallVector<int64_t>> convertDenseIntAttr(
+    Optional<DenseIntElementsAttr> optionalAttr, Optional<Location> loc) {
```

Comment:
It doesn't sound right to me that something with a generic name like `convertDenseIntAttr` would be picky about the shape of its argument. Perhaps rename to `convert1DAttribute`?